.github/workflows/benchmark_master.yml

```diff
@@ -0,0 +1,77 @@
+# Workflow to trigger/schedule benchmarking, cleaning, aggregating on master branch only and storing results in S3
+name: Benchmark Master Branch
+on:
+  push:
+    branches: ["master"]
+  schedule:
+    - cron: '0 0 * * 0'
+
+env:
+  AG_MODULE: tabular
+  AG_PRESET: medium
```

Comment:
We might want to discuss with the team 1. how frequent we want to run the benchmark 2. what preset we want to compare (running all presets are expensive) 3. what benchmark we want to run (current just test, which is dummy). For now this is good

Reply:
Got it, will add it as a parking lot for today